README.md

```diff
@@ -259,6 +259,37 @@ declare module 'fastify' {
 }
 ```
 
+## Provider Quirks
+
+The following providers require additional work to be set up correctly.   
+
+### Twitch
+
+Twitch requires that the request for a token in the oauth2 flow contains the `client_id` and `client_secret` property.
+Unfortunately, this library cannot set this up automatically. When authenticating via Twitch, include the
```

Comment:
"Unfortunately"? Is there a possibility to set this up automatically? I dont think so. So I would remove that sentence with unforrtunately.

@Fdawgs You are a native speaker of traditional english. Wdyt?

Reply:
Still valid, it is unfortunate it can't be setup automatically. 😊